k8s/openebs-operator.yaml

```diff
@@ -152,6 +158,29 @@ spec:
         - name: MAYA_PORTAL_URL
           value: "https://mayaonline.io/"
 ---
+kind: Deployment
+apiVersion: extensions/v1beta1
```

Comment:
Should apps be used instead of extensions?

Reply:
Makes sense. Probably with 0.6 we can move towards "apps" for all the deployments listed here. Right @AmitKumarDas?

Reply:
chnaged to `apps`